oauth2client/file_storage.py

```diff
@@ -32,34 +31,18 @@ class CredentialsFileSymbolicLinkError(Exception):
     """Credentials files must not be symbolic links."""
 
 
-class Storage(BaseStorage):
+class FileStorage(BaseStorage):
     """Store and retrieve a single credential to and from a file."""
 
     def __init__(self, filename):
         self._filename = filename
-        self._lock = threading.Lock()
 
     def _validate_file(self):
         if os.path.islink(self._filename):
             raise CredentialsFileSymbolicLinkError(
                 'File: %s is a symbolic link.' % self._filename)
 
-    def acquire_lock(self):
```

Comment:
Not 100% sure yet, but the way this is currently done `File` is no longer threadsafe, but doing:

```python
storage = LockedStorage(FileStorage())
```

Would be.